test/light-dom.html

```diff
@@ -0,0 +1,172 @@
+<!doctype html>
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+  <script src="../../web-component-tester/browser.js"></script>
+
+  <link rel="import" href="common.html">
+  <link rel="import" href="../iron-lazy-pages.html">
+  <link rel="import" href="../../iron-selector/iron-selector.html">
+</head>
+<body>
+
+<test-fixture id="light">
```

Comment:
This test file is almost equivalent to the other one. Therefore I would like to not copy-paste all tests, but merge the two and only let the fixture be different. Maybe wrap all tests in a function and call this twice, for each fixture you created.